tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react-jsx"
```

Comment:
That was changed automatically. Keep it?

Reply:
I noticed this happening to me and I kept not committing it, thinking it's something with my install :). Let's leave it then.